packages/server/test/hooks.js

```diff
@@ -39,11 +39,22 @@ exports.truncateTables = async (tableNames) => {
         .whereRaw("tablename not like '%knex%'")
         .whereNotIn('tablename', protectedTables)
     ).map((table) => table.tablename)
-  }
 
-  if (!tableNames.length) return
+    // We're deleting everything, so lets turn off triggers to avoid deadlocks/slowdowns
+    await knex.transaction(async (trx) => {
```

Comment:
I was getting random deadlocks, I think the cascade call was getting too heavy, especially if all triggers are enabled